grpc_opa/authorizer.go

```diff
@@ -193,12 +195,15 @@ func (a *DefaultAuthorizer) Evaluate(ctx context.Context, fullMethod string, grp
 		return false, ctx, fmt.Errorf("%q", errs)
 	}
 
+	reqID, _ := requestid.FromContext(ctx)
```

Comment:
Instead of empty "" request_id if it doesn't exist, should we instead use something like "no_request_id"?  @drewwells any comments on this?
```
reqID, ok := requestid.FromContext(ctx)
if !ok {
  reqID = "no_request_id"
}
```

Reply:
Updated